src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -245,6 +247,7 @@ int GetNdkApiLevel(string androidNdkPath, string androidApiLevel, AndroidTargetA
 			LogDebugMessage ("  SupportedAbis: {0}", SupportedAbis);
 			LogDebugTaskItems ("  ResolvedAssemblies:", ResolvedAssemblies);
 			LogDebugTaskItems ("  AdditionalNativeLibraryReferences:", AdditionalNativeLibraryReferences);
+			LogDebugTaskItems ("  Profiles:", Profiles);
```

Comment:
I've been removing log messages like these. Since we no longer support XBuild, MSBuild will end up logging these twice, since it already logs every input.

Reply:
OK, I guess the diagnostic output should be enough. Maybe we should remove all such prints in our tasks?

Reply:
I've just removed them as I changed a task. But maybe a big PR removing them all is worth it?